src/test/java/com/moodysalem/TimeZoneMapperTest.java

```diff
@@ -75,25 +73,19 @@ public String getExpectedTimezone() {
     public void runTestCases() {
         final AtomicInteger succeeded = new AtomicInteger(0), failed = new AtomicInteger(0);
 
-        final Consumer<TestData> tester = (d) -> {
-            String timezone = TimezoneMapper.tzNameAt(d.getLat(), d.getLng());
-            ZoneId zone = TimezoneMapper.tzAt(d.getLat(), d.getLng());
+        for (TestData data: testData) {
+            String timezone = TimezoneMapper.tzNameAt(data.getLat(), data.getLng());
 
-            String error = String.format(EXPECT_ERROR, d.getName(), d.getLat(), d.getLng(), d.getExpectedTimezone(), timezone);
+            String error = String.format(EXPECT_ERROR, data.getName(), data.getLat(), data.getLng(),
+                    data.getExpectedTimezone(), timezone);
             try {
-                assertTrue(error, d.getExpectedTimezone().equals(timezone));
-                assertTrue(error, ZoneId.of(d.getExpectedTimezone()).equals(zone));
+                assertTrue(error, data.getExpectedTimezone().equals(timezone));
                 succeeded.incrementAndGet();
             } catch (AssertionError e) {
                 failed.incrementAndGet();
                 LOG.warning(e.getMessage());
             }
-        };
-
-        testData.forEach(tester);
-
-        // hand entered test cases
-        tester.accept(new TestData("Chicago, IL", 41.8369, -87.6847, "America/Chicago"));
```

Comment:
Done. I tried to use TestNG's `successPercentage` annotation, but it only works for repeated invocations of the method for the same set of arguments. Unfortunately for us, it does not gate success based off of all of the times the test method is called. That's why I decided to use before/after suite methods.